components/google-cloud/google_cloud_pipeline_components/experimental/custom_job/custom_job.py

```diff
@@ -24,7 +24,7 @@
 from kfp.components import structures
 
 _DEFAULT_CUSTOM_JOB_MACHINE_TYPE = 'n1-standard-4'
-_DEFAULT_CUSTOM_JOB_CONTAINER_IMAGE = 'gcr.io/managed-pipeline-test/gcp-launcher:v7'
+_DEFAULT_CUSTOM_JOB_CONTAINER_IMAGE = 'gcr.io/managed-pipeline-test/gcp-launcher:latest'
```

Comment:
should we pin to a specific version, even for the master branch?

Reply:
Good point, my thought here is that as we do development on `master` we can push out changes to the container without having to constantly up the version and do a corresponding code change in this file.  However that means `latest` tag is always unstable which is not optimal,  instead we could keep `latest` pinned to the last release, `v0.1.4` in this case and change the `master` branch to the actual branch version , which would be `v0.1.5.dev` in the current setting. 

I like the second approach, I will submit a PR accordingly we can decide on that PR to stay with status quo or change as mentioned above.